tests/emscripten_main_loop.cpp

```diff
@@ -66,8 +66,12 @@ void looper() {
     assert(ret == 0);
   } else if (frame == 90) {
     emscripten_cancel_main_loop();
-    emscripten_set_main_loop(looper, 100, 1);
+    emscripten_set_main_loop(looper, 110, 1);
   } else if (frame == 100) {
+    emscripten_pause_main_loop();
+    emscripten_cancel_main_loop();
+    emscripten_set_main_loop(looper, 110, 1);
```

Comment:
Although there are are several "main loop" tests, I feel they test slightly different things, and also they use the "usual" style of coding.

I believe testing this issue in only this particular .cpp file suffices in this case.